jcdb-core/src/main/kotlin/org/utbot/jcdb/impl/types/signature/JvmTypes.kt

```diff
@@ -72,6 +121,11 @@ open class JvmTypeVariable(val symbol: String) : JvmType() {
     override val displayName: String
         get() = symbol
 
+    override fun setNullability(nullable: Boolean): JvmType =
+        JvmTypeVariable(symbol, nullable).also {
+            it.declaration = declaration
+        }
+
     override fun equals(other: Any?): Boolean {
```

Comment:
Am I right that both types with `nullable = true` and `nullable=false` will be equal?

Reply:
Deleted `equals` and `hashCode` completely as legacy